exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggBatch.java

```diff
@@ -263,13 +343,13 @@ private HashAggregator createAggregatorInternal() throws SchemaChangeException,
 
       // add this group-by vector to the output container
       groupByOutFieldIds[i] = container.add(vv);
+      columnMapping.put(outputField.getName(), ne.getRef().getAsNamePart().getName());
```

Comment:
The regressions *TestHashAggEmitOutcome* set the "name" and the "expr" to different strings, hence the failure above. So I changed this to:
```
      columnMapping.put(outputField.getName(), ne.getExpr().toString().replace('`',' ').trim() );
```
To make the regressions (e.g. **testHashAggrMultipleEMITOutcome()**) pass.
It is a little ugly (removing the '`' before and after), but I could not think of something nicer.